src/heartbeat.ts

```diff
@@ -144,7 +144,8 @@ export class HeartbeatManager extends Disposable {
             }, this.logger);
         } catch (e) {
             const suffix = wasClosed ? 'closed heartbeat' : 'heartbeat';
-            this.logger.error(`Failed to send ${suffix}, triggered by ${this.lastActivityEvent} event:`, e);
+            e.message = `Failed to send ${suffix}, triggered by event: ${this.lastActivityEvent}: ${e.message}`;
```

Comment:
- Wrap error message
- Use `this.logger.error(e)` to avoid error message in collected logs shows `failed to xxx: {protocol: http}`, it expect Error as its first argument

Reply:
<img width="1432" alt="image" src="https://user-images.githubusercontent.com/20944364/225552146-d1d12ce7-a9bf-4294-942c-d870398edf5f.png">